.github/workflows/template-build.yml

```diff
@@ -0,0 +1,57 @@
+on:
+  workflow_call:
+    inputs:
+      release-tag-name:
+        required: true
+        type: string
+      push:
+        required: true
+        type: boolean
+env:
+  imageName: "rancher/harvester-seeder"
+
+jobs:
+  dapper-build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Run dapper
+        run: make ci
+
+      - name: Read some Secrets
+        uses: rancher-eio/read-vault-secrets@main
+        if: ${{ inputs.push == true }}
+        with:
+          secrets: |
+            secret/data/github/repo/${{ github.repository }}/dockerhub/rancher/credentials username | DOCKER_USERNAME ;
+            secret/data/github/repo/${{ github.repository }}/dockerhub/rancher/credentials password | DOCKER_PASSWORD
```

Comment:
Not sure whether we need to change `rancher` to `harvester`, but we can merge this first and fix it if need.

https://github.com/harvester/terraform-provider-harvester/blob/4e07ed02cba6fe7fc37bc0d41ff433df81e69915/.github/workflows/build.yml#L58-L59

Reply:
`rancher` is the correct org